### PR TITLE
Update messages

### PR DESCRIPTION
**Pull request by Pix1234**

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	"createwiki-error-notalnum": "The database name may not contain non-alphanumeric characters.",
 	"createwiki-error-notlowercase": "The database name may not contain uppercase letters.",
 	"createwiki-error-notsuffixed": "The database name doesn't end in a valid suffix (e.g. 'wiki').",
-	"createwiki-error-usernotcreated": "The requester user was not created on the wiki.",
+	"createwiki-error-usernotcreated": "The user that requested the wiki, was not created on the wiki.",
```
Comment (MacFan4000) [inline i18n/en.json]: There should probs be a space between to and the

Comment (Pix1234) [reply i18n/en.json]: Huh?

File: i18n/en.json [hunk 2]
```diff
@@ -25,29 +25,29 @@
 	"createwiki-label-submit": "Submit",
 	"createwiki-legend": "Create a wiki",
 	"createwiki-success": "The wiki was successfully created.",
-	"log-description-farmer": "This log tracks changes to wiki settings.",
+	"log-description-farmer": "This log tracks changes tothe wiki's settings.",
```
Comment (MacFan4000) [inline i18n/en.json]: There should probs be a space between to and the.

File: i18n/en.json [hunk 3]
```diff
@@ -25,29 +25,29 @@
 	"createwiki-label-submit": "Submit",
 	"createwiki-legend": "Create a wiki",
 	"createwiki-success": "The wiki was successfully created.",
-	"log-description-farmer": "This log tracks changes to wiki settings.",
+	"log-description-farmer": "This log tracks changes to the wiki's settings.",
```
Comment (Reception123) [inline i18n/en.json]: maybe "this" wiki's settings would be better

File: i18n/en.json [hunk 4]
```diff
@@ -25,29 +25,29 @@
...
 	"logentry-farmer-requestwiki": "$1 requested the wiki \"$4\" (language: $5; private: $6) in $7",
 	"log-name-farmer": "Farmer log",
 	"requestwiki": "Request a wiki",
-	"requestwiki-error-csrf": "<strong>Sorry, the request could not be submitted due to a loss of session data.</strong>\nPlease submit the form again. If that doesn't work, please [[Special:Userlogout|log out]] and log back in.",
-	"requestwiki-error-invalidcomment": "Your wiki request contains an invalid character. Please try requesting again, and if the problem persists [[Help center|please contact us]]",
-	"requestwiki-error-nodomain": "You must specify either a subdomain or custom domain.",
+	"requestwiki-error-csrf": "<strong>Sorry, the request could not be submitted due to session data loss.</strong>\nPlease submit the form again. If that doesn't work, please [[Special:Userlogout|log out]] and log back in.",
+	"requestwiki-error-invalidcomment": "Your wiki request contains an <strong>invalid</strong> character. Please try again, and if the problem persists [[Help center|please contact us]]",
+	"requestwiki-error-nodomain": "You must input a subdomain or custom domain.",
```
Comment (Reception123) [inline i18n/en.json]: I think "specify either" is more clear in this case, and we should keep it.

File: i18n/en.json [hunk 5]
```diff
@@ -25,29 +25,29 @@
...
 	"requestwiki-label-language": "Language:",
 	"requestwiki-label-private": "Private wiki (not readable by everyone):",
 	"requestwiki-label-sitename": "Wiki name:",
 	"requestwiki-label-siteurl": "Subdomain:",
-	"requestwiki-notloggedin": "Sorry, you need to <span class=\"plainlinks\">[$1 log in]</span> before you can request a wiki.",
+	"requestwiki-notloggedin": "Sorry, you must <span class=\"plainlinks\">[$1 log in]</span> before you can request a wiki.",
 	"requestwiki-submit": "Submit",
 	"requestwiki-success": "Your wiki request ($1) was successfully submitted. Your request will be reviewed by a volunteer as soon as possible.",
-	"requestwikiqueue": "Wiki requests queue",
+	"requestwikiqueue": "Wiki requests",
```
Comment (Reception123) [inline i18n/en.json]: This is the queue for wiki requests, so we should probably keep the word "queue" in it to avoid any confusion.

**Comment by Pix1234** [comment]
Done @Reception123

**Comment by paladox** [comment]
needs rebasing.

**Pull request closed without merging**